FBSimulatorControl/Processes/FBProcessQuery.m

```diff
@@ -223,7 +223,7 @@ - (NSArray *)processesWithLaunchPathSubstring:(NSString *)substring
     if (!info) {
       return YES;
     }
-    if ([info.launchPath rangeOfString:substring].location == NSNotFound) {
+    if (substring && [info.launchPath rangeOfString:substring].location == NSNotFound) {
```

Comment:
I'm happy to do this, but you should update the header documentation `@param substring the substring that must exist in the launch path.` to include that omiting the param will return all processes.